Sources/ManageServer.php

```diff
@@ -255,7 +255,7 @@ function AlignURLsWithSSLSetting($new_force_ssl = 0)
 	require_once($sourcedir . '/Subs-Admin.php');
 
 	// Check $boardurl
-	if ($new_force_ssl == 2)
+	if ($new_force_ssl > 0)
```

Comment:
Is empty in any kind better as this solution?

Reply:
Run small test ~ 1mio calls the different is small,
when you do 10 mio calls "> 0" is faster then empty().
empty: 3.9425780773163
> 0: 3.2624399662018

~ 20% faster (but empty() and > 0 are fast already)

Reply:
We need to use `empty()` because the variable might not be set at all. Once we've done `empty()`, there is no reason to also test whether it 0 or greater than 0.

Reply:
You're sure that can happen? https://github.com/albertlast/SMF2.1/blob/a9bd146a16bcdcd866500aa82f6905148df843ef/Sources/ManageServer.php#L252
But like i said the diff between the solution is margin so i don't care which way is choosen.